src/ripple/overlay/impl/Manifest.cpp

```diff
@@ -275,6 +276,17 @@ ManifestDisposition
 ManifestCache::applyManifest (
     Manifest m, ValidatorList& unl, beast::Journal journal)
 {
+    /*
+        Move master public key from permanent trusted key list
+        to manifest cache.
+    */
+    if (unl.trusted(m.masterKey))
```

Comment:
Nit: Put the declaration of comment in the if so we don't have to lookup twice: `if (auto comment = unl.member (m.masterKey))`